addons/website/static/src/builder/plugins/options/portal_option_plugin.js

```diff
@@ -0,0 +1,53 @@
+import { Plugin } from "@html_editor/plugin";
+import { registry } from "@web/core/registry";
+import { withSequence } from "@html_editor/utils/resource";
+import { StyleAction } from "@html_builder/core/core_builder_action_plugin";
+
+class PortalOptionPlugin extends Plugin {
```

Comment:
```suggestion
class PortalOptionPlugin extends Plugin {
```

Let's have plugin name more specific: `PortalCardOptionPlugin`?